monitoring/rid_qualifier/host/run_locally.sh

```diff
@@ -37,6 +37,7 @@ docker run --name rq-worker -d --rm \
   -e MOCK_HOST_DSS_URL="${DSS}" \
   -e REDIS_URL=redis://redis-server:6379/0 \
   -v `pwd`/build/test-certs:/var/test-certs:ro \
+  -v /tmp:/mnt/app/input-files \
```

Comment:
Do we need to mount a volume here?  Mounting a volume enables the container to 1) access files pre-existing on the host system and 2) create or modify files on the host system.  Do we need either of these things here?  If not, we can drop the volume mount at runtime (this argument), and instead add a `RUN mkdir -p /mnt/app/input-files` (or any other directory name) in the Dockerfile to create a folder that exists only within the container.  If 1 above is needed, what files should pre-exist in /tmp and where have we documented that?  If 2 above is needed, what files are produced/modified by rid_qualifier host and where is that documented?